halyard-web/pkg_scripts/postInstall.sh

```diff
@@ -5,13 +5,21 @@ echo '/opt/halyard/bin/hal "$@"' | sudo tee -a /usr/local/bin/hal > /dev/null
 
 chmod +x /usr/local/bin/hal
 
-HAL_USER=""
```

Comment:
Shell is weird. 
Regardless of this removal `HAL_USER` will be available to `install ... --owner=$HAL_USER`.
This is because the variables are effectively global in bash unless defined in a function using `local` keyword.